modules/govuk_jenkins/templates/jobs/smokey.yaml.erb

```diff
@@ -30,9 +30,30 @@
           auth-token: <%= @environment_variables['SECOND_LINE_SLACK_AUTH_TOKEN']%>
           build-server-url: <%= @slack_build_server_url %>
           room: <%= @slack_room %>
-      - email:
+      - email-ext:
           recipients: govuk-ci-notifications@digital.cabinet-office.gov.uk
-          send-to-individuals: true
+          failure: true
+          presend-script: |
```

Comment:
@surminus I'm not sure I understand how to get that into the YAML file? (This is where we reach the limit of my Puppet understanding).  Although I've noticed we're already writing raw shell script on lines 59-59 in this file as well anyway...